cmd/livepeer/livepeer.go

```diff
@@ -159,7 +160,13 @@ func main() {
 	orchWebhookURL := flag.String("orchWebhookUrl", "", "Orchestrator discovery callback URL")
 	detectionWebhookURL := flag.String("detectionWebhookUrl", "", "(Experimental) Detection results callback URL")
 
-	flag.Parse()
+	// Config file (used instead of flags)
```

Comment:
Is this comment accurate? IIUC the flags are still used if they are specified and they take precedence over the config file if that is specified as well.

Reply:
Right, updated.